celery/backends/redis.py

```diff
@@ -218,16 +218,19 @@ def __init__(self, host=None, port=None, db=None, password=None,
         if host and '://' in host:
             url, host = host, None
 
-        self.max_connections = (
-            max_connections or
-            _get('redis_max_connections') or
-            self.max_connections)
+        if max_connections is not None:
+            self.max_connections = max_connections
+        elif _get('redis_max_connections') is not None:
+            self.max_connections = _get('redis_max_connections')
+        else:
+            self.max_connections = self.max_connections
         self._ConnectionPool = connection_pool
 
         socket_timeout = _get('redis_socket_timeout')
         socket_connect_timeout = _get('redis_socket_connect_timeout')
         retry_on_timeout = _get('redis_retry_on_timeout')
         socket_keepalive = _get('redis_socket_keepalive')
+        socket_keepalive_options = self._transport_options.get('socket_keepalive_options', {})
```

Comment:
should we document this change?

Reply:
You have [this](https://docs.celeryq.dev/en/stable/userguide/configuration.html#result-backend-transport-options) description for those parameters and the rest parameters are in code they aren't described either.
So I haven't described it as well. So the question is: should I be highlighted?

Reply:
at least New in version 5.4. annotation?